### ISSUE TITLE
Verify skidding/friction

### ISSUE THREAD

**Issue by mikepurvis** [g/grizzly_simulator#3]
There needs to be some more rigorous comparisons made between simulated and real Grizzly, in order to verify the dynamic parameters.

**Comment by pmukherj** [comment]
Agreed. Also on the Husky. They are overestimated currently. Even if its not a straight numerical comparison (aka system identification) just to scale it back to match basic driving experience would be worth it.

**Comment by rgariepy** [comment]
There's also a certain degree in uncertainty wrt. if Gazebo handles such
calculations properly. There's no proper tire model, just a friction model.

**Comment by usam1249** [comment]
https://github.com/g/grizzly_simulator/issues/3#issue-37083109
